### PR TITLE
[AETHER-1265] Integrate Java profiler in TOST env

### PR DESCRIPTION
**Pull request by pierventre**
Add a nodeport for each onos instance to enable the connection between client and profiler agent.

### CODE REVIEW

File: onos-classic/values.yaml [hunk 1]
```diff
@@ -31,6 +31,11 @@ resources:
 
 individualOpenFlowNodePorts: false
 
+# Adds an additional nodeport to allow profiler connection.
+# When false, profiler can still connect through proxy/port-forward
+profiler:
+  enabled: false
```
Comment (charlesmcchan) [inline onos-classic/values.yaml:37]: The name is confusing. Profiler is always enabled when using profiler image. What we control here is whether to further expose a nodePort or not. I'd suggest renaming this to `nodePort: true`

**Review by hwchiu — APPROVED** [review]
Same concern as Charles said and the rest are looks good to me.

File: onos-classic/values.yaml (hunk 1 above)
Comment (charlesmcchan) [reply onos-classic/values.yaml:37]: Or even more specific: `exposeNodePort: true`

**Review by charlesmcchan — APPROVED** [review]

**Review by kuujo — APPROVED** [review]

**Comment by pierventre** [comment]
retest this please

**Comment by Andrea-Campanella** [comment]
@ray-milkey is there a way we can setup a travis job to deploy the newly updated charts from the patch and at least check that a 3 node onos with 3 node atomix reaches running state in k8s ? this woudl be helpful for TOAST and for VOLTHA teams. How do you test helm chart changes in SD-RAN ? @sauravdas2

**Comment by pierventre** [comment]
we have satisfied the reqs - can someone with the power merge this ?

**Pull request merged**